{
  "id": "MAL-2024-4752",
  "summary": "Malicious code in ailzyn1tr0 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2024-06-25T13:32:17Z",
  "published": "2024-06-25T13:32:17Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2024-06-25T13:32:17Z",
        "import_time": "2024-06-28T02:48:14.414533643Z",
        "versions": [
          "1.0"
        ],
        "id": "RLMA-2024-03532",
        "source": "reversing-labs",
        "sha256": "eec39cb13b5ff9189983c87d108eb1a4191b0ce516787834e816e254601a38e0"
      }
    ]
  },
  "references": [
    {
      "type": "WEB",
      "url": "https://github.com/DataDog/malicious-software-packages-dataset"
    }
  ],
  "affected": [
    {
      "package": {
        "name": "ailzyn1tr0",
        "ecosystem": "PyPI",
        "purl": "pkg:pypi/ailzyn1tr0"
      },
      "versions": [
        "1.0"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/ailzyn1tr0/MAL-2024-4752.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}